runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonPlugin.java

```diff
@@ -77,6 +81,8 @@
 	private CannonCounter counter;
 	private boolean skipProjectileCheckThisTick;
 	private boolean cannonBallNotificationSent;
+	private WorldPoint clickedCannonLocation;
+	private boolean firstCannonLoad = false;
```

Comment:
`false` is the default initialized value for fields.
```suggestion
	private boolean firstCannonLoad;
```